src/ui/components/Main/component.ts

```diff
@@ -240,6 +244,58 @@ export default class extends Component {
   toggleVisualizer() {
     this.isVisualizerShowing = !this.isVisualizerShowing;
   }
+
+  hideShare() {
+    this.shareIsVisible = false;
+
+    if(this.shareTrigger) {
+      setTimeout(function(trigger) {
+        trigger.focus();
+      }, 100, this.shareTrigger);
```

Comment:
🤕 

Experiencing some problems with timing of the `willDestroy` hook in `OverlayKeyBindings` tearing down an event listener.